content/ngf/installation/installing-ngf/deploy-data-plane.md

```diff
@@ -0,0 +1,243 @@
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
+cafe-nginx   NodePort    10.96.125.117   <none>        80:30180/TCP   5m2s
+```
+
+The Service type can be changed, explained in the next section.
+
+## How to modify provisioned NGINX instances
+
+The NginxProxy custom resource can modify the provisioning of the Service object and NGINX deployment when a Gateway is created.
+
+{{< note >}} Updating most Kubernetes related fields in NginxProxy will trigger a restart of the related resources. {{< /note >}}
+
+An NginxProxy resource is created by default after deploying NGINX Gateway Fabric. This NginxProxy resource is attached to the GatewayClass (created on NGINX Gateway Fabric deployment), and 
+its settings are applied globally to all Gateways.
+
+Use `kubectl get` and `kubectl describe` to get some more information on the resource:
+
+```shell
+kubectl get nginxproxies -A   
+```
+```text                                                                                                                               ⎈ kind-kind
+NAMESPACE       NAME                      AGE
+nginx-gateway   ngf-proxy-config   19h
+```
+
+```shell
+kubectl describe nginxproxy -n nginx-gateway ngf-proxy-config
+```
+```text                                                                                                ⎈ kind-kind
+Name:         ngf-proxy-config
+Namespace:    nginx-gateway
+Labels:       app.kubernetes.io/instance=mngf
+              app.kubernetes.io/managed-by=Helm
+              app.kubernetes.io/name=nginx-gateway-fabric
+              app.kubernetes.io/version=edge
+              helm.sh/chart=nginx-gateway-fabric-1.6.2
+Annotations:  meta.helm.sh/release-name: ngf
+              meta.helm.sh/release-namespace: nginx-gateway
+API Version:  gateway.nginx.org/v1alpha2
+Kind:         NginxProxy
+Metadata:
+  Creation Timestamp:  2025-05-05T23:01:28Z
+  Generation:          1
+  Resource Version:    2245
+  UID:                 b545aa9e-74f8-45c0-b472-f14d3cab936f
+Spec:
+  Ip Family:  dual
+  Kubernetes:
+    Deployment:
+      Container:
+        Image:
+          Pull Policy:  IfNotPresent
+          Repository:   nginx-gateway-fabric/nginx
+          Tag:          edge
+      Replicas:         1
+    Service:
+      External Traffic Policy:  Local
+      Type:                     NodePort
+Events:                         <none>
+```
+
+From the information obtained with `kubectl describe` you can see the default settings for the provisioned NGINX Deployment and Service.
+Under `Spec.Kubernetes` you can see a few things:
+- The NGINX container image settings
+- How many NGINX Deployment replicas are specified
+- The type of Service and external traffic policy
+
+{{< note >}} Depending on installation configuration, the default NginxProxy settings may be slightly different from what is shown in the example. {{< /note >}}
```

Comment:
I think it would be worth linking to the API doc around here somewhere. Right now the phrasing here makes it seem like these are the only available fields.